src/test/java/uk/gov/hmcts/darts/task/runner/impl/AudioLinkingAutomatedTaskTest.java

```diff
@@ -107,46 +102,48 @@ class EventProcessorTest {
 
         private AudioLinkingAutomatedTask.EventProcessor eventProcessor;
 
+        @Mock
+        private MediaLinkedCaseHelper mediaLinkedCaseHelper;
+
         @BeforeEach
         void beforeEach() {
+
             this.eventProcessor = spy(
                 new AudioLinkingAutomatedTask.EventProcessor(
-                    mediaRepository, hearingRepository, mediaLinkedCaseRepository,
-                    eventService, userAccountRepository, Duration.ofSeconds(0)
+                    mediaRepository,
+                    eventService, userAccountRepository, mediaLinkedCaseHelper,
+                    Duration.ofSeconds(0)
                 )
             );
             lenient().when(userAccountRepository.getReferenceById(SystemUsersEnum.AUDIO_LINKING_AUTOMATED_TASK.getId())).thenReturn(userAccount);
         }
 
         @Test
-        void positiveProcessEvent() {
-            doNothing().when(eventProcessor)
-                .processMedia(any(), any(), any());
+        void processEvent_typical() {
```

Comment:
Suggest we align to the new [test naming standard](https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1694706043#BestPractices(Backend)-TestNamingShallClearlyCommunicatetheTestIntent).